cmd/swarmctl/cluster/inspect.go

```diff
@@ -52,6 +53,28 @@ func printClusterSummary(cluster *api.Cluster) {
 			fmt.Fprintf(w, "    %s: %s\n", ca.Protocol, ca.URL)
 		}
 	}
+
+	if cluster.Spec.DefaultLogDriver != nil {
+		fmt.Fprintf(w, "DefaultLogDriver\t: %s\n", cluster.Spec.DefaultLogDriver.Name)
```

Comment:
I still think it's weird to have one line that uses the field name, and all the others are descriptions.